### PR TITLE
Adding kabedon keyboards to VIA

### PR DESCRIPTION
**Pull request by 370490639**
"Adding kabedon keyboards to VIA,Both are tested working with the via software.
Thanks!"
kd98
78s

## Checklist

- [x] The QMK source code follows the guide here: https://caniusevia.com/docs/configuring_qmk
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by Wilba6582** [comment]
This needs a QMK PR.

**Comment by 370490639** [comment]
> 这需要QMK PR。
thank，I'm doing it
https://github.com/qmk/qmk_firmware/pull/9929

**Comment by 370490639** [comment]
@Wilba6582 
QMK PR is done,
its already in the master branch.
https://github.com/qmk/qmk_firmware/pull/9929

File: src/other/kabedon/kd78s/kabedon78s.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  "matrix": { "rows": 6, "cols": 18 },
+  "layouts": {
+    "labels": [
+      "ROWS : F1~F12",
+      "COLS : G1~G5" ,
+      "PAD : 21",
+      "Split : Pad +",
+      "Split : Pad Enter",
+      "Split : Pad 0"
+    ],
```
Comment (Wilba6582) [inline src/other/kabedon/kd78s/kabedon78s.json:44]: It is not very clear what these do. Perhas they should be "Function Row", "Macro Columns", "Extra Numpad Keys", "Split Numpad +", "Split Numpad Enter", "Split Numpad Zero".

**Comment by Wilba6582** [comment]
Also, please correct the colors of keys, there is a row which is "mod" colored not "alpha" colored:

image

**Comment by 370490639** [comment]
@Wilba6582 
Thank you for your tips, I have corrected.

**Pull request merged**